sdk/trace/span_processor_test.go

```diff
@@ -109,8 +109,20 @@ func TestUnregisterSpanProcessorWhileSpanIsActive(t *testing.T) {
 	}
 }
 
-// TODO(rghetia): Add Shutdown test when it is implemented.
-func TestShutdown(t *testing.T) {
+func TestSpanProcessorShutdown(t *testing.T) {
+	name := "Increment shutdown counter of a span processor"
+	sp := NewTestSpanProcessor()
+	if sp == nil {
+		t.Fatalf("Error creating new instance of TestSpanProcessor\n")
+	}
+
+	wantCount := sp.shutdownCount + 1
+	sp.Shutdown()
+
+	gotCount := sp.shutdownCount
+	if wantCount != gotCount {
+		t.Errorf("%s: wrong counter: got %d, want %d\n", name, gotCount, wantCount)
+	}
```

Comment:
maybe test Shutdown using Register/Unregister. Multiple Unregister should not trigger multiple shutdown call.

Reply:
But actually it triggers. I added another test demonstrating this.
Should I create an issue to fix this behavior?

Reply:
yes, please.

Reply:
Also, you can link the issue in a comment, so that whoever addresses the issue definitively knows that this test breaking is expected

Reply:
I created a new issue: https://github.com/open-telemetry/opentelemetry-go/issues/175
I can fix it, I'm going to add a new sync.Once variable to span_processor.go